onnx_tf/frontend.py

```diff
@@ -294,6 +303,38 @@ def tensorflow_graph_to_onnx_graph(cls,
         initializer=consts_proto,
         value_info=value_info_proto)
 
+  @classmethod
+  def _data_type_caster(cls, protos, data_type_cast_map):
+    """Cast to a new data type if node name is in data_type_cast_map.
+    Be used to precess protos to match ONNX type constraints.
+
+    :param protos: Target protos.
+      TensorProto for inputs and ValueInfoProto for consts.
+    :param data_type_cast_map: A {node.name: new_data_type} dict.
+    :return: Processed protos.
+    """
+    if not data_type_cast_map:
+      return protos
+    result = []
+    for proto in protos:
+      new_proto = proto
+      if proto.name in data_type_cast_map \
```

Comment:
Is it possible to remove this \ and replace it with (, ) around two conditions?

My past experience working on TF shows that TF team doesn't like \.